drivers/hmis/app/models/hmis/hud/current_living_situation.rb

```diff
@@ -14,7 +14,6 @@ class Hmis::Hud::CurrentLivingSituation < Hmis::Hud::Base
   include ::Hmis::Hud::Concerns::ServiceHistoryQueuer
   include ::Hmis::Hud::Concerns::HasCustomDataElements
 
-  belongs_to :enrollment, **hmis_enrollment_relation, optional: true
```

Comment:
Why are you removing these? It's probably fine, but these (CLS, Disability, Employment Education, etc.) are all tied to enrollments.  Maybe I just missed the concern that's now getting pulled in somewhere?

Reply:
It was moved into the ClientProjectEnrollmentRelated mixin. It was necessary to move the enrollment association before the project association as it's now `has_one :project, through: :enrollment` and order matters in that case